src/inmanta/data_pg.py

```diff
@@ -1850,10 +1850,10 @@ def done(self) -> int:
         limit_statement = f"LIMIT {limit} " if limit is not None and limit > 0 else ""
         offset_statement = f"OFFSET {offset} " if offset is not None and offset > 0 else ""
         query = f"SELECT c.*, SUM(CASE WHEN r.status NOT IN({transient_states}) THEN 1 ELSE 0 END) AS done ," + \
```

Comment:
This is rather sloppy because it uses + one some lines and on others not. I prefer as concise as possible so no + and no \

Lets try to apply this to all new and updated queries/strings